wpimath/src/main/native/include/frc/kinematics/SwerveDriveOdometry.h

```diff
@@ -30,20 +30,6 @@ namespace frc {
 template <size_t NumModules>
 class SwerveDriveOdometry {
  public:
-  /**
-   * Constructs a SwerveDriveOdometry object.
-   *
-   * @param kinematics The swerve drive kinematics for your drivetrain.
-   * @param gyroAngle The angle reported by the gyroscope.
-   * @param wheelPositions The wheel positions reported by each module.
-   * @param initialPose The starting position of the robot on the field.
-   */
-  template <typename... ModulePositions>
-  SwerveDriveOdometry(SwerveDriveKinematics<NumModules> kinematics,
-                      const Rotation2d& gyroAngle,
-                      ModulePositions&&... wheelPositions,
-                      const Pose2d& initialPose = Pose2d{});
```

Comment:
It'll work if you put the variadic at the end.

Reply:
Yea. Since it's simple enough to wrap the module positions in `{}`, I removed this constructor. It might be valuable to teams to document that you can do that, though.